app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from __future__ import annotations
```

Comment:
please remove this import

Reply:
When I remove it I have an error in annotations
<img width="798" alt="image" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/10523657/2b49945a-6c34-4a1a-9f09-ecdc3f5202a3">

Reply:
Ah, got it )